index.html

```diff
@@ -1,13 +1,208 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <!-- font style  -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,900;1,100;1,300;1,400;1,700&display=swap"
+    rel="stylesheet">
+
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
+
+
   <link rel="stylesheet" href="style.css">
   <title>WeLead conferences</title>
 </head>
+
 <body>
+  <main>
```

Comment:
For accessibility reasons, I think it is semantically appropriate that your page be divided into the `header`, `main`, and `footer` sections. currently, you have wrapped the header and footer inside the main tag, I suggest you use the `main` tag for your main content after your header and before your footer😉😉😉😉 Kindly implement same for your about page too🤗🤗🤗

Reply:
okay thanks